test/SlowTests/Cluster/ShardedClusterTransactionTests.cs

```diff
@@ -0,0 +1,131 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Runtime.CompilerServices;
+using System.Threading.Tasks;
+using Raven.Client.Documents;
+using Raven.Client.Documents.Session;
+using Raven.Client.ServerWide;
+using Sparrow.Utils;
+using Tests.Infrastructure;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace SlowTests.Cluster
+{
+    public class ShardedClusterTransactionTests : ClusterTransactionTests
+    {
+        public ShardedClusterTransactionTests(ITestOutputHelper output) : base(output)
+        {
+        }
+
+        protected override DocumentStore GetDocumentStore(Options options = null, [CallerMemberName] string caller = null)
```

Comment:
revert

Reply:
Don't you like overriding the `GetDocumentStore`?
The alternative is to have a function test for sharded and unsharded with `DocumentStore` as a parameter.
But that means I need to create the cluster and the document store separately for sharded and unsharded tests.
I think the "override" is cleaner? Or did I miss the meaning of your comment?